README.md

```diff
@@ -1,18 +1,34 @@
-# web-prototype
+# React + TypeScript + Vite
```

Comment:
Eventually would be good to put instructions for how to build for dummies (i.e. me)

Reply:
added commands for development/build